Makefile

```diff
@@ -9,14 +9,16 @@ COPY_DEREFERENCED=cp -RH
 RM_FILE=rm -f
 RM_DIR=rmdir -p --ignore-fail-on-non-empty
 
-ETCDIR_NX   ?= /etc/nxagent
-PREFIX      ?= /usr/local
-BINDIR      ?= $(PREFIX)/bin
-LIBDIR      ?= $(PREFIX)/lib
-USRLIBDIR   ?= $(LIBDIR)
-INCLUDEDIR  ?= $(PREFIX)/include
-NXLIBDIR    ?= $(LIBDIR)/nx
-CONFIGURE   ?= ./configure
+ETCDIR_NX       ?= /etc/nxagent
+PREFIX          ?= /usr/local
+BINDIR          ?= $(PREFIX)/bin
+LIBDIR          ?= $(PREFIX)/lib
+SHLIBDIR        ?= $(LIBDIR)
+NXLIBDIR        ?= $(SHLIBDIR)/nx
+USRLIBDIR       ?= $(NXLIBDIR)/X11
+USRLIBDIRPATH   ?= $(USRLIBDIR)
```

Comment:
We probably don't need to override `USRLIBDIRPATH` if `USRLIBDIR` gets properly overridden. Have you tried building this branch? Is the RPATH set correctly? (Also respecting multilib.)

If yes, and leaving out `USRLIBDIRPATH` in this Makefile also builds and leads to the correct result, we can drop it here for simplicity.